tensorflow_lite_support/python/task/text/bert_nl_classifier.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Bert NL Classifier task."""
+
+import dataclasses
+from typing import Optional
+
+from tensorflow_lite_support.python.task.core import base_options as base_options_module
+from tensorflow_lite_support.python.task.processor.proto import classifications_pb2
+from tensorflow_lite_support.python.task.text.pybinds import _pywrap_bert_nl_classifier
+
+_CppBertNLClassifier = _pywrap_bert_nl_classifier.BertNLClassifier
+_BaseOptions = base_options_module.BaseOptions
+
+
+@dataclasses.dataclass
+class BertNLClassifierOptions:
+  """Options for the Bert NL classifier task."""
```

Comment:
Please add docstring to explain each parameter. They will show up in API documentation like this:
https://www.tensorflow.org/lite/api_docs/python/tflite_support/task/audio/AudioClassifierOptions

I realized that some option classes don't have parameter docstring so please send a follow-up PR to fix other classes.